examples/memstore-plugin/main.go

```diff
@@ -45,7 +46,9 @@ func main() {
 	opts := memory.Options{}
 	opts.InitFromViper(v)
 
-	grpc.Serve(&memoryStore{store: memory.NewStore()})
+	grpc.Serve(&config.PluginServices{
+		Store: &memoryStore{store: memory.NewStore()},
```

Comment:
I think it would be advisable to implement archive/capabilities in the memstore-plugin, as a way of testing the changes in the grpc layer.